source/tutorial/verify-mongodb-packages.txt

```diff
@@ -10,34 +10,39 @@ Verify Integrity of MongoDB Packages
    :depth: 1
    :class: singlecol
 
-Overview
---------
-
 The MongoDB release team digitally signs all software packages to
 certify that a particular MongoDB package is a valid and unaltered
-MongoDB release. Before installing MongoDB, you should validate the package
-using either the provided PGP signature or SHA-256 checksum.
+MongoDB release. Before installing MongoDB, you should validate the 
+package using either the provided PGP signature or SHA-256 checksum.
 
 PGP signatures provide the strongest guarantees by checking both the
 authenticity and integrity of a file to prevent tampering.
 
 Cryptographic checksums only validate file integrity to prevent network
 transmission errors.
 
-Procedures
-----------
+Verify Linux/macOS Packages
+---------------------------
 
 Use PGP/GPG
-+++++++++++
+~~~~~~~~~~~
 
-MongoDB signs each release branch with a different PGP key. The public key files
-for each release branch since MongoDB 2.2 are available for download
-from the `key server <https://www.mongodb.org/static/pgp/>`_ in both textual
-``.asc`` and binary ``.pub`` formats.
+MongoDB signs each release branch with a different PGP key. The public 
+key files for each release branch since MongoDB 2.2 are available for
+download from the `key server <https://www.mongodb.org/static/pgp/>`_ 
+in both textual ``.asc`` and binary ``.pub`` formats.
 
 .. include:: /includes/steps/install-verify-files-pgp.rst
 
 Use SHA-256
-+++++++++++
+~~~~~~~~~~~
 
 .. include:: /includes/steps/install-verify-files-sha.rst
+
+Verify Windows Packages
+-----------------------
+
+This verifies the MongoDB binary against its SHA256 key. This tutorial 
+uses the latest release of MongoDB Community Edition 3.4.
```

Comment:
Why version 3.4 and not a more recent version?

Reply:
That's what was there originally and seemed as good a place as any. (Also, I think it's what's used in the Linux example.)

The results I show in the example are true; I actually tested them.

Reply:
Ok. Would it make sense to put a note saying that the following examples use MongoDB 3.4 but the procedure works with later versions (or all versions, or list specific versions) of MongoDB?